packages/backend/package.json

```diff
@@ -39,11 +39,14 @@
     "pg": "^8.7.3",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "rollbar": "^2.25.2",
     "source-map-support": "^0.5.21",
-    "zod": "^3.17.3",
-    "rollbar": "^2.25.2"
+    "starknet": "^4.4.2",
+    "zod": "^3.17.3"
   },
   "devDependencies": {
+    "@types/bn.js": "^5.1.1",
+    "@types/elliptic": "^6.4.14",
```

Comment:
Had to add these two, because it was not automatically added when installing `starknet`.